lib/util.js

```diff
@@ -103,6 +104,19 @@ util = {
                     return callback(_.set(e, 'help', `the url "${location}" did not provide valid JSON data`));
                 }
 
+                var error,
+                    resource;
+
+                if ((response.statusCode !== 200) || (body && body.error)) {
```

Comment:
only stop on non 200 code but, if body.error is present, use it only to extract message.